sources/platform/integrations/airbyte.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Airbyte
+sidebar_lbel: Airbyte
+description: Learn how to integrate your Apify Datasets with Airbyte.
+sidebar_position: 11.15
+slug: /integrations/airbyte
+---
+
+**Learn how to integrate your Apify Datasets with Airbyte.**
+
+---
+
+Airbyte is an open-source data integration platform that allows you to move your data between different sources and destinations using pre-built connectors, which are maintained either by Airbyte itself or by its community.
+One of these connectors is the Apify Dataset connector, which makes it simple to move data from Apify Datasets to any supported destination.
```

Comment:
```suggestion
One of these connectors is the Apify dataset connector, which makes it simple to move data from Apify datasets to any supported destination.
```

Reply:
I fixed the second occurence of Apify dataset , but I feel like first should be left capitalized as it is the name of the connector itself within Airbyte.